### ISSUE TITLE
Add Brown (Red) Noise sound

### ISSUE THREAD

**Issue by rafaelmardojai** [rafaelmardojai/blanket#112]
Moved from #50

**Comment by osirase** [comment]
Brown noise is best noise :)

**Comment by rafaelmardojai** [comment]
I was sent this sound by Greg Farough [brown.zip](https://github.com/rafaelmardojai/blanket/files/8986662/brown.zip), he said he could synthesize a new one if we wanted a different length, bitrate, etc.

CC @Porrumentzio @haggen88

**Comment by zalsader** [comment]
I would love to see this

**Comment by haggen88** [comment]
@rafaelmardojai I'm going to look for some loops...

**Comment by haggen88** [comment]
- 'brown noise.wav'  by NoiseCollector [CC BY 3.0]   // [NoiseCollector -brown noise.wav.ogg.zip](https://github.com/rafaelmardojai/blanket/files/11362685/NoiseCollector.-brown.noise.wav.ogg.zip)
from https://freesound.org/people/NoiseCollector/sounds/3941/


- 'Brown Noise 20 Hz to 20 KHz 10 Seconds.wav' by reaktorplayer [CC BY-NC 4.0] // [reaktorplayer - Brown Noise 20 Hz to 20 KHz 10 Seconds.wav.ogg.zip](https://github.com/rafaelmardojai/blanket/files/11362700/reaktorplayer.-.Brown.Noise.20.Hz.to.20.KHz.10.Seconds.wav.ogg.zip)
from https://freesound.org/people/reaktorplayer/sounds/94220/


- 'Brown Noise 10 seconds' by MarcosPurziani [CC0 1.0] // [MarcosPurziani - Brown Noise 10 seconds.ogg.zip](https://github.com/rafaelmardojai/blanket/files/11362702/MarcosPurziani.-.Brown.Noise.10.seconds.ogg.zip)
from  https://freesound.org/people/MarcosPurziani/sounds/537286/

@rafaelmardojai  I edited the length of the audio files to avoid certain cuts in the loops. I hope that @Porrumentzio  can analyze them according to the parameters that Blanket handles.

**Comment by diaconic** [comment]
@haggen88 I don't believe we'd need a sample. Audacity lets you generate brown noise from within the app.

![image](https://github.com/rafaelmardojai/blanket/assets/22648124/5fbe884b-23a1-4502-afc6-c1cb0a04ce4c)

It doesn't have any clicks or seams when looping or stopping abruptly. I'm assuming this is suitable.

**Comment by katalia369** [comment]
::repost from duplicate @ https://github.com/rafaelmardojai/blanket/issues/283::
White noise and pink noise are already on the system, Red Noise (also called Brown Noise) is just another slightly more bass variant. I happen to like red noise more than the other two. I have a generator at home but I can't use that in the office so I run Blanket on the laptop there.

I have reasonable audio equipment at home for client virtual communications. I used that and my generator to pull a two-minute .ogg file. I put down two quick icon files in black and white. I am skilled enough to make a recording but not remotely close to understanding the levels of sound mentioned in the guide so I am posting the files here in hopes someone with more skill can build on this base.

[red.noise.tar.gz](https://github.com/rafaelmardojai/blanket/files/13850335/red.noise.tar.gz)

**Comment by kurbkick** [comment]
I love this app and would also appreciate it if brown noise was added as an option.
